express/blocks/shared/carousel.js

```diff
@@ -63,10 +63,10 @@ export function buildCarousel(selector = ':scope > *', $parent, infinityScrollEn
     toggleControls();
   };
   $arrowLeft.addEventListener('click', () => {
-    moveCarousel(240);
+    moveCarousel($platform.offsetWidth);
```

Comment:
Caution: @So1ene is modifying the same code in #437 but slightly differently. You may want to align here...

Reply:
I did this, I prefer the solution in https://github.com/adobe/express-website/pull/437